packages/rancher-vsphere/rancher-vsphere-cpi/generated-changes/patch/questions.yaml.patch

```diff
@@ -0,0 +1,26 @@
+--- charts-original/questions.yaml
++++ charts/questions.yaml
+@@ -40,23 +40,3 @@
+     type: string
+     group: Configuration
+     show_if: "vCenter.credentialsSecret.generate=false"
+-
+-  - variable: vCenter.labels.generate
```

Comment:
Seeing a bunch of new patches removing bits from the chart; is that intentional?

Reply:
No, looks like the patch undid a lot of the changes I need... after talking to @snasovich I apparently also need to bump to supercharge version 100.1, because 100.0 is already being released for 2.7.0. I'll make those changes and re push to the branch